package.json

```diff
@@ -29,7 +29,7 @@
     "docs": "node ./scripts/docs/packages.js",
     "bundle": "node ./scripts/docs/bundle.js",
     "lint": "jshint scripts/ packages/ system-test/ test/ && jscs packages/ system-test/ test/",
-    "test": "npm run docs && mocha test/docs.js packages/*/test/*.js",
+    "test": "npm run docs && npm run bundle && mocha test/docs.js packages/*/test/*.js",
```

Comment:
This might not work as you intend it to. The tricky thing about bundling the docs together is that what is in the `master` branch of `google-cloud` could be a slightly older version of any one of the dependencies  (e.g. `@google-cloud/bigquery`).

That being said when I wrote the bundle task I had to consider using one of the following approaches:

1. Using a submodule to checkout a tag for each dependency and re-running the docs against said tag.
2. Copying pre-existing JSON and modifying keys to contain code specific to their package.

I ended up going with option 2. That being said, when you bundle, it assumes there's a ghpages submodule that it can find the appropriate version of any `@google-cloud` packages and copies them.

All that being said, I don't think this task will do what you're hoping it does :<